tests/UI/specs/UIIntegration_spec.js

```diff
@@ -530,7 +530,7 @@ describe("UIIntegrationTest", function () { // TODO: Rename to Piwik?
     it('should fail correctly when db information in config is incorrect', function (done) {
         testEnvironment.configOverride = {
             database: {
-                host: '127.50.50.50',
+                host: 'localhost',
```

Comment:
That makes sense, just need to make sure the error message doesn't say 127.50.50.50 cannot be found (since this error doesn't seem to happen on travis)